.ci/no-exception-test.sh

```diff
@@ -5,6 +5,12 @@ removeFolderWithProtectedFiles() {
   find $1 -delete
 }
 
+function getCheckstylePomVersion {
+  echo "$(mvn -e --no-transfer-progress -q -Dexec.executable='echo' \
+                      -Dexec.args='${project.version}' \
+                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)"
```

Comment:
small refactoring - util function to avoid long boilerplate lines in getting checkstyle version